config/development.json

```diff
@@ -37,13 +37,8 @@
 		"ad-tracking": false,
 		"automated-transfer": true,
 		"apple-pay": true,
+		"boosted-post-survey": true,
 		"code-splitting": true,
-		"comments/filters-in-posts": true,
-		"comments/moderation-tools-in-posts": true,
-		"comments/management": true,
-		"comments/management/all-list": false,
-		"manage/comments/bulk-actions": false,
```

Comment:
Please be careful when removing flags that are not related to the PR @dllh @akirk